tests/test_vnet.py

```diff
@@ -8,1222 +8,7 @@
 from swsscommon import swsscommon
 from pprint import pprint
 from dvslib.dvs_common import wait_for_result
-
-
-def create_entry(tbl, key, pairs):
```

Comment:
Why are these removed? Can we add few tests for BGP routes?

Reply:
Its not removed. I just moved all of the helper functions and helper class into a separate file vnet_lib. This way they can be reused by different test files. Currently we have 30 tests in the test_vnet and running it very slow. Adding mroe thests to would make it mroe slow. I mvoed the helper code to the  vnet_lib to slim down this file. 

I have added the BGP and VNET route coexistence tests in the file test_vnet2.py if that is what you mean by tests for BGP routes.